src/proto/TransactionCompiler.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+
+package TW.TxCompiler.Proto;
```

Comment:
Despite the length of the source file name, let's use `TransactionCompiler` in the namespace as well if possible.

Reply:
In that case it could be prefixed, but it's not important.